packages/compartment-mapper/test/language-for-extension.test.js

```diff
@@ -0,0 +1,111 @@
+import 'ses';
+import test from 'ava';
+import { scaffold } from './scaffold.js';
+import jsonp from './_parse-jsonp.js';
+import text from '../src/parse-text.js';
+
+// The JSONP parser uses harden, as a bit.
+lockdown({
+  errorTaming: 'unsafe',
+  errorTrapping: 'none',
+});
+
+scaffold(
```

Comment:
For reviewers new to Compartment Mapper, `scaffold` generates a combination of test cases where a single fixture is expected to produce the same behavior for multiple flows:

- executed directly off a file system
- loaded off the file system then later executed
- archived off the file system, then parsed, then executed
&c